### ISSUE TITLE
Error message is sent from scylla_run not scylla

### ISSUE THREAD

**Issue by tzach** [scylladb/scylla#821]
Unlike other traces, the following error is reported as  `scylla_run`, not `scylla`

```
Jan 19 10:55:11 ip-172-30-0-13 scylla_run[9270]: Exiting on unhandled exception of type 'exceptions::configuration_exception': Saved cluster name CassandraCluster != configured name Tzach
```

@avikivity [suspect](https://groups.google.com/forum/#!searchin/scylladb-dev/scylla_run%7Csort:relevance%7Cspell:false/scylladb-dev/M5nuWP3O0MA/G-JXhVt5EwAJ) this is a result of using printf instead of logger

To reproduce:
1. stop scylla
2. change the `cluster_name` in scylla.yaml
3. start scylla
4. journalctl -xe

**Comment by avikivity** [comment]
@syuu1228 we should just remove scylla_run, it doesn't do much, instead execute scylla directly.

**Comment by syuu1228** [comment]
Can we drop all $NETWORK_MODE related script / config, make it just for posix for now?
Since we specified User as scylla, I think it already does not work with dpdk mode.

**Comment by avikivity** [comment]
We can change NETWORK_MODE to NETWORK_ARGS and write something like this:

    # uncomment for DPDK
    #NETWORK_ARGS= --network-stack native --dpdk-pmd

    # uncommend for virtio/vhost
    #NETWORK_ARGS= --network-stack native

**Comment by syuu1228** [comment]
Sure, but DPDK requires root permission, and Unit file seems doesn't accept environment variable for User= parameter, so we still not able to run DPDK mode.

**Comment by avikivity** [comment]
Does it really require root, or a set of capabilities?  If the latter, we can use the configuration setting Capabilities= to give the process the needed capabilities.

I think that these days nothing in the kernel checks for root, rather the checks are for permissions or CAP_WHATEVER, so we can use capabilities, and a pre-init service to set permissions.

**Comment by slivne** [comment]
scylla is not run via a scylla_run anymore - so this error will not be logged from scylla_run but from scylla

commit f33656ef0367f1d6b8ac54fcafdce41b18ef54ce
Author: Takuya ASADA <syuu@scylladb.com>
Date:   Thu Jan 28 00:58:30 2016 +0900

    dist: eliminate startup script
    
    Fixes #373
    
    Signed-off-by: Takuya ASADA <syuu@scylladb.com>
    Message-Id: <1453910311-24928-1-git-send-email-syuu@scylladb.com>


Leaving this open till we have a solution for the DPDK case

**Comment by avikivity** [comment]
Gone in a5bb6c4b1b65819fbc3531dcbf70c9e8f593616e, closing.
